### ISSUE TITLE
Error with debootstrap: cant open it because it says "no such file or directory".

### ISSUE THREAD

**Issue by Knightlycat** [dnschneid/crouton#3836]
hronos@localhost / $ sudo sh -e ~/Downloads/crouton -r kali-rolling -t xfce
Downloading latest crouton installer...
################################################################################################################################################ 100.0%
Installing kali-rolling-amd64 chroot to /usr/local/chroots/kali-rolling
################################################################################################################################################ 100.0%
Patching debootstrap...
Downloading bootstrap files...
E: You must install wget to download packages.
debootstrap error log:
tail: cannot open '/tmp/crouton.kbW/kali-rolling-amd64/debootstrap/debootstrap.log' for reading: No such file or directory
Failed to run debootstrap.

I tried this after my previous problem in issue #3830. says Wget needed to download things, and for some reason it just cant open the debootstrap. 

Basically, I think its sending the debootstrap to some other directory and is not checking that same directory.

**Comment by i2blue** [comment]
is it possible to run sudo sh -e ~/Downloads/crouton -r kali-rolling -t gnome for like the gnome desktop enviorment

**Comment by Knightlycat** [comment]
I really wouldn't know, you would have to try it. BUT I MUST WARN YOU, for me when i finish my command, i have to enter chroot and delete a specific file. just look it up

**Comment by i2blue** [comment]
i have and its like everytime i think i find the answer to it it isn the answer lol ima try it

**Comment by Knightlycat** [comment]
@i2blue hope it works.

**Comment by dnschneid** [comment]
Hopefully it's fixed now...

**Comment by carylwyatt** [comment]
I'm glad to see that the debootstrap download issue seems to be cleared up, but I can't figure out why I'm still getting a debootstrap error.

I don't have crouton installed on this machine, so I was trying to run the "quick test" you mentioned above with `sudo sh ~/Downloads/crouton -r xenial -d -f xenial.bs` but it gets stuck after downloading everything and trying to move on to extraction.

```
I: Retrieving zlib1g 1:1.2.8.dfsg-2ubuntu4
I: Validating zlib1g 1:1.2.8.dfsg-2ubuntu4
I: Chosen extractor for .deb packages: ar
I: Extracting adduser...
debootstrap error log:
                                 Dload  Upload   Total   Spent    Left  Speed
100 51328  100 51328    0     0  46705      0  0:00:01  0:00:01 --:--:-- 47350
ar: error while loading shared libraries: libbfd-2.30.so: cannot open shared object file: No such file or directory
Failed to run debootstrap.
```

I've looked through other issues, but this "no such file or directory" one seems the closest to my problem.

Thanks!
